environment/src/main/java/com/sequenceiq/environment/environment/poller/FreeIpaPollerProvider.java

```diff
@@ -118,4 +120,39 @@ private boolean freeipaAvailable(DescribeFreeIpaResponse freeipa) {
     private boolean freeIpaUsersSynchronized(SyncOperationStatus freeIpaResponse) {
         return SynchronizationStatus.COMPLETED.equals(freeIpaResponse.getStatus());
     }
+
+    public AttemptResult<Void> upgradeCcmPoller(Long envId, String envCrn, String operationId) {
+        if (PollGroup.CANCELLED.equals(EnvironmentInMemoryStateStore.get(envId))) {
+            LOGGER.info("FreeIpa polling cancelled in inmemory store, id: " + envId);
+            return AttemptResults.breakFor("FreeIpa polling cancelled in inmemory store, id: " + envId);
+        }
+        OperationStatus operationStatus = freeIpaService.getOperationStatus(operationId);
+        if (operationCompleted(operationStatus)) {
```

Comment:
Can we have a debug log about the operation status?
I think it would be handy in case of debugging.

Reply:
ok